components/script/dom/abstractworkerglobalscope.rs

```diff
@@ -131,8 +132,8 @@ pub fn run_worker_event_loop<T, TimerMsg, WorkerMsg, Event>(
         // Batch all events that are ready.
         // The task queue will throttle non-priority tasks if necessary.
         match task_queue.try_recv() {
-            None => match timer_event_port.try_recv() {
-                None => match devtools_port.and_then(|port| port.try_recv()) {
+            None => match timer_event_port.try_recv().ok() {
+                None => match devtools_port.and_then(|port| port.try_recv().ok()) {
```

Comment:
Nit: Remove those two `.ok()`s and change:

* `Some` to `Ok`
* `None` to `Err(_)`